builds/misc/images-release.yaml

```diff
@@ -0,0 +1,602 @@
+jobs: 
+################################################################################
+  - job: linux_dotnet_projects
+################################################################################
+    displayName: LinuxDotnet
+    pool: 
+      name: "Hosted Ubuntu 1604"
+    steps: 
+      - bash: |
+            docker login '$(build.registry.address)' --username '$(build.registry.user)' --password '$(build.registry.password)'
+            docker login '$(registry.address)' --username '$(registry.user)' --password '$(registry.password)'
+        displayName: 'Docker Login'
+      - task: Bash@3
+        displayName: "Install Rust"
+        inputs: 
+          arguments: "--package-arm"
+          filePath: ./edgelet/build/linux/install.sh
+          targetType: filePath
+      - bash: "echo \"##vso[task.setvariable variable=PATH;]$HOME/.cargo/bin:$PATH\""
+        displayName: "Modify path"
+      - template: ../templates/install-dotnet3.yaml
+      - bash: "cargo install --git https://github.com/arsing/cross.git --branch set-path"
+        displayName: "Install cross (fork with docker fix)"
+      - task: ShellScript@2
+        displayName: "Build Azure-IoT-Edge-Core"
+        inputs: 
+          args: "-c Release"
+          scriptPath: scripts/linux/buildBranch.sh
+      - task: DotNetCoreCLI@2
+        displayName: "Functions Binding nuget package"
+        inputs: 
+          buildProperties: OutDir=$(Build.BinariesDirectory)/publish/Microsoft.Azure.WebJobs.Extensions.EdgeHub
+          command: pack
+          nobuild: true
+          packDirectory: $(Build.BinariesDirectory)/publish/
+          packagesToPack: "**/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj"
+          versionEnvVar: version
+          versioningScheme: byEnvVar
+      - task: DotNetCoreInstaller@0
```

Comment:
Why use 2.1? Please add comment to explain?

Reply:
We need dotnet 2.1 for the code sign job. I tried with dotnet 3.1 and it will not work.

Reply:
MS hosted agent should already install dotnet 2.1, did you try run without this step?

Reply:
I wasn't able to get this to work without installing dotnet 2.1 explicitly. If dotnet 3.1 is the last version installed it will fail the code sign.

Reply:
Thanks for trying out.  Please add comments here.  thanks.